tools/virtiostat_example.txt

```diff
@@ -0,0 +1,47 @@
+Demonstrations of virtiostat, the Linux eBPF/bcc version.
+
+
+This program traces virtio devices to analyze the IO operations and
+throughput.
+
+Example
+#./virtiostat -d 10
+Tracing virtio devices statistics ... Hit Ctrl-C to end.
+--------
+          Driver       Device      VQ Name       In SGs      Out SGs            In BW           Out BW
+      virtio_net      virtio0     output.2            0            9                0              832
+      virtio_net      virtio0     output.3            0           92                0            13053
```

Comment:
Let us ensure default output has at most 80 char's per line. Here we exceeded it.